solutions/ibm-catalog/stack/sap-s4hana-bw4hana-demo/stack_definition.json

```diff
@@ -0,0 +1,336 @@
+{
+  "inputs": [
+    {
+      "name": "powervs_zone",
+      "required": true,
+      "type": "string",
+      "hidden": false
+    },
+    {
+      "name": "power_infrastructure_prefix",
+      "required": true,
+      "type": "string",
+      "hidden": false,
+      "default": "infra",
+      "custom_config": {}
+    },
+    {
+      "name": "external_access_ip",
+      "required": true,
+      "type": "string",
+      "hidden": false,
+      "default": "0.0.0.0/0",
+      "custom_config": {}
+    },
+    {
+      "name": "client_to_site_vpn",
+      "type": "object",
+      "hidden": false,
+      "default": {
+        "enable": true,
+        "client_ip_pool": "192.168.0.0/16",
+        "vpn_client_access_group_users": []
+      },
+      "custom_config": {}
+    },
+    {
+      "name": "ssh_public_key",
+      "required": true,
+      "type": "password",
+      "hidden": false,
+      "custom_config": {}
+    },
+    {
+      "name": "ssh_private_key",
+      "required": true,
+      "type": "password",
+      "hidden": false,
+      "custom_config": {}
+    },
+    {
+      "name": "sap_system_prefix",
+      "required": true,
+      "type": "string",
+      "hidden": false,
+      "default": "sapsys",
+      "custom_config": {}
+    },
+    {
+      "name": "powervs_hana_instance_sap_profile_id",
+      "required": true,
+      "type": "string",
+      "hidden": false,
+      "default": "ush1-4x256",
+      "custom_config": {}
+    },
+    {
+      "name": "ibmcloud_cos_service_credentials",
+      "required": true,
+      "type": "password",
+      "hidden": false,
+      "custom_config": {}
+    },
+    {
+      "name": "ibmcloud_cos_configuration",
+      "required": true,
+      "type": "object",
+      "hidden": false,
+      "default": {
+        "cos_region": "eu-geo",
+        "cos_bucket_name": "powervs-automation",
+        "cos_hana_software_path": "HANA_DB/rev78",
+        "cos_solution_software_path": "S4HANA_2023",
+        "cos_swpm_mp_stack_file_name": ""
+      },
+      "custom_config": {}
+    },
+    {
+      "name": "sap_solution",
+      "required": true,
+      "type": "string",
+      "hidden": false,
+      "custom_config": {}
+    }
+  ],
+  "members": [
+    {
+      "inputs": [
+        {
+          "name": "prerequisite_workspace_id",
+          "value": "ref:../Power%20Virtual%20Server%20with%20VPC%20landing%20zone/outputs/schematics_workspace_id"
+        },
+        {
+          "name": "powervs_zone",
+          "value": "ref:../../inputs/powervs_zone"
+        },
+        {
+          "name": "prefix",
+          "value": "ref:../../inputs/sap_system_prefix"
+        },
+        {
+          "name": "powervs_sap_network_cidr",
+          "value": "10.53.0.0/24"
```

Comment:
shouldn't this come from user ? why is it hardcoded here ? What happens when a user gives a different value ? Are you using the values correctly ?